src/compiler/emitter.ts

```diff
@@ -6891,12 +6891,13 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                 if (currentSourceFile.text.charCodeAt(comment.pos + 1) === CharacterCodes.asterisk) {
                     return currentSourceFile.text.charCodeAt(comment.pos + 2) === CharacterCodes.exclamation;
                 }
+                // Omit /// comment if compilerOptions.removeComments is true.
                 // Verify this is /// comment, but do the regexp match only when we first can find /// in the comment text
                 // so that we don't end up computing comment string and doing match for all // comments
-                else if (currentSourceFile.text.charCodeAt(comment.pos + 1) === CharacterCodes.slash &&
+                if (!compilerOptions.removeComments && (currentSourceFile.text.charCodeAt(comment.pos + 1) === CharacterCodes.slash &&

```

Comment:
I do not think this is the right fix. the method is called `isPinnedOrTripleSlashComment` yet it is isPennedOrSomeTimesTripleSlashComments. 

The question is why are we emitting tripple slash comments.. other than the top of the file, why do we care?